src/components/Entry.vue

```diff
@@ -225,6 +248,14 @@ export default {
     };
   },
   computed: {
+    /** Returns if the authentication type includes date of birth */
+    isInputDateOfBirth() {
```

Comment:
This also seems a bit off to me.. a variable and a method for all the things. There should be a more direct relation with the group parameters... But we can look at it together later